### ISSUE TITLE
[BUG] "reporters" is an unrecognized parameter vimure() (R interface)

### ISSUE THREAD

**Issue by marioangst** [latentnetworks/vimure#80]
**Describe the bug**
The documentation of the R package indicates the possibility to specify the set of reporters: https://latentnetworks.github.io/vimure/latest/tutorials/R/tutorial02-introduction-to-vimure.html#implicit-assumptions

However, passing reporters results in a warning: **vm.model.VimureModel      - WARNING - Ignoring unrecognised parameter reporters.** 

**To Reproduce**
vimure(edgelist, reporters = unique(edgelist$ego))

**Expected behavior**
An error message in case of malspecification, else the reporter mask to be adequately set to the reporter set

**Environment (please complete the following information):**
 - OS: Windows 11
 - R/Python versions: 4.1/ 3.6

**Comment by marioangst** [comment]
The parameter is also unrecognized using the Python module on its own, eg. through 
`model.fit(edgelist, reporters=list(set(edgelist.ego)))`
(I was guessing a list might be required here)

**Comment by jonjoncardoso** [comment]
Hey there, @marioangst! Thanks for flagging this issue. Your dedication to testing the package is much appreciated!

After a closer look, I realised that the warning is indeed accurate and that the root cause of the issue is our outdated documentation. We had this parameter in the past but removed it after noticing that using a list of reporters isn't clear enough and could lead to confusion. Questions like "Which ties were the reporters asked about?" or "Were they only asked about ties they were involved in?" could arise. 

The reporters' mask (the `R` parameter) is currently the only way to inform reporters and how they were allowed to report ties. 

The Python and R tutorials were edited to reflect this (https://github.com/latentnetworks/vimure/commit/a56ac35bd46f680b8d701ef8b070b99612ad167e), and I'm closing this issue.  Feel free to add to it and re-open it if you feel necessary.
